### DISCUSSION TITLE
STMicro

### DISCUSSION THREAD

**Discussion by tracyphasespace** [will127534/IMU_Array#2] [Ideas]
Have you considered the STMicro IMUs?
https://www.st.com/en/mems-and-sensors/lsm6dsv16x.html

**Comment by will127534** [comment]
Yes, because it is the one the original paper uses.
But I'm not using them because [ICM-42688-P](https://invensense.tdk.com/products/motion-tracking/6-axis/icm-42688-p/) has  32Khz Clock input options for all the IMUs to use a single and stable clock source.

ST can only do sampling synchronization, but not external clocking.

**Reply by tracyphasespace** [reply]
I will check. I'm pretty sure we did it with a single stable clock.
